docs/src/inputs/input/devSandbox/devSandbox.jsx

```diff
@@ -0,0 +1,565 @@
+                        Error with custom message
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Indicates that the input has an error by passing an string with a message.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputErrorWithText').default}
+                >
+                    <ExampleInputErrorWithText />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        Fluid
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        An input can take on the size of its container.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputFluid').default}
+                >
+                    <ExampleInputFluid />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        Icon
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        An input can be formatted with an icon.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputIcon').default}
+                >
+                    <ExampleInputIcon />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        Icon (Custom JSX)
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        An input can be formatted with an icon. You can either supply a string or add a custom icon by passing a node.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/ExampleInputIconCustom').default}
+                >
+                    <ExampleInputIconCustom />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        Inverse
+                    </Heading>
```

Comment:
`anchorLink` should be something like `"inverse"`

Reply:
Fixed. Please check this [Commit](https://github.com/saddlebackdev/react-cm-ui/pull/353/commits/46c752aa0e781aedba48ebab9e7a168431118aa2)

Reply:
Looks good.  Resolving.